modules/installation-gcp-marketplace.adoc

```diff
@@ -4,45 +4,39 @@
 
 :_content-type: PROCEDURE
 [id="installation-gcp-marketplace_{context}"]
-= Using a GCP Marketplace image
-If you want to deploy an {product-title} cluster using a GCP Marketplace image, you must create the manifests and edit the compute machine set definitions to specify the GCP Marketplace image.
+= Using the GCP Marketplace offering
 
-.Prerequisites
-
-* You have the {product-title} installation program and the pull secret for your cluster.
+Using the GCP Marketplace offering lets you deploy an {product-title} cluster, which is billed on pay-per-use basis (hourly, per core) through GCP, while still being supported directly by Red{nbsp}Hat.
 
-.Procedure
+By default, the installation program downloads and installs the {op-system-first} image that is used to deploy compute machines. To deploy an {product-title} cluster using an {op-system} image from the GCP Marketplace, override the default behavior by modifying the `install-config.yaml` file to reference the location of GCP Marketplace offer.
 
-. Generate the installation manifests by running the following command:
-+
-[source,terminal]
-----
-$ openshift-install create manifests --dir <installation_dir>
-----
+.Prerequisites
 
-. Locate the following files:
+* You have an existing `install-config.yaml` file.
 
-** `<installation_dir>/openshift/99_openshift-cluster-api_worker-machineset-0.yaml`
-** `<installation_dir>/openshift/99_openshift-cluster-api_worker-machineset-1.yaml`
-** `<installation_dir>/openshift/99_openshift-cluster-api_worker-machineset-2.yaml`
+.Procedure
 
-. In each file, edit the `.spec.template.spec.providerSpec.value.disks[0].image` property to reference the offer to use:
+. Edit the `compute.platform.gcp.osImage` parameters to specify the location of the GCP Marketplace image:
+** Set the `project` parameter to `redhat-marketplace-public`.
+** Set the `name` parameter to one of the following offerings:
 +
-{product-title}:: `projects/redhat-marketplace-public/global/images/redhat-coreos-ocp-48-x86-64-202210040145`
-{opp}:: `projects/redhat-marketplace-public/global/images/redhat-coreos-opp-48-x86-64-202206140145`
-{oke}:: `projects/redhat-marketplace-public/global/images/redhat-coreos-oke-48-x86-64-202206140145`
+{product-title}:: `redhat-coreos-ocp-48-x86-64-202210040145`
```

Comment:
According to the [KCS](https://access.redhat.com/articles/6976804), the images like `redhat-coreos-{ocp|opp|oke}-48` are for 4.12 or earlier, and 4.13+ should use `redhat-coreos-{ocp|opp|oke}-413` ones. Thanks!